pkg/ccl/backupccl/manifest_handling.go

```diff
@@ -36,27 +36,37 @@ import (
 	"github.com/cockroachdb/errors"
 )
 
+// Files that may appear in a backup directory.
 const (
-	// BackupManifestName is the file name used for serialized
-	// BackupManifest protos.
+	// BackupManifestName is the file name used for serialized BackupManifest
+	// protos.
 	BackupManifestName = "BACKUP"
-	// BackupNewManifestName is a future name for the serialized
-	// BackupManifest proto.
+	// BackupNewManifestName is a future name for the serialized BackupManifest
+	// proto.
 	BackupNewManifestName = "BACKUP_MANIFEST"
 
 	// BackupPartitionDescriptorPrefix is the file name prefix for serialized
 	// BackupPartitionDescriptor protos.
 	BackupPartitionDescriptorPrefix = "BACKUP_PART"
-	// BackupManifestCheckpointName is the file name used to store the
-	// serialized BackupManifest proto while the backup is in progress.
+	// BackupManifestCheckpointName is the file name used to store the serialized
+	// BackupManifest proto while the backup is in progress.
 	BackupManifestCheckpointName = "BACKUP-CHECKPOINT"
+	// BackupStatisticsFileName is the file name used to store the serialized
+	// table statistics for the tables being backed up.
+	BackupStatisticsFileName = "BACKUP-STATISTICS"
+	// BackupSentinelWriteFile is a file that we write to the backup directory to
+	// ensure that we have write privileges to the directory. Nothing should check
+	// for its existence since we don't guarantee that it's cleaned up after it is
+	// written (for example, we may not have DELETE permissions for the
+	// destination, which should be allowed).
+	BackupSentinelWriteFile = "BACKUP-SENTINEL-PREPARE"
```

Comment:
nit: I'd maybe add `cockroach` to the name since that might be what you get in a "permission denied" error log, and I'm not sure if either `prepare` or sentinel` are meaningful to an end-user. 

maybe `COCKROACH-BACKUP-PLACEHOLDER` ? I donno.

Reply:
I like `COCKROACH-BACKUP-PLACEHOLDER` more, can go with that one now. It should be fairly easy to change in the future especially considering we shouldn't be reading it.